sqe-http-api/Services/ArtefactService.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Google.Protobuf.WellKnownTypes;
```

Comment:
What's this for? Why do we even use Protobufs?

Reply:
I have absolutely no idea...  It isn't used by anything, so I removed it.